test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/PaginationSample.cs

```diff
@@ -5,51 +5,115 @@ namespace Tizen.NUI.Samples
 {
     public class PaginationSample : IExample
     {
+        private TextLabel board1, board2;
         private Pagination pagination1;
         private Pagination pagination2;
+        private View root;
+        private View parent1, parent2;
 
         private readonly int PAGE_COUNT = 5;
 
         public void Activate()
         {
             Window window = NUIApplication.GetDefaultWindow();
 
+            root = new View()
+            {
+                Size = new Size(1920, 1080),
+                BackgroundColor = new Color(0.7f, 0.9f, 0.8f, 1.0f),
+            };
+            window.Add(root);
+            window.KeyEvent += Window_KeyEvent;
+
+            parent1 = new View()
+            {
+                Position = new Position(500, 400),
+                Size = new Size(700, 70)
+            };
+            var parent1layout = new LinearLayout();
+            parent1layout.LinearOrientation = LinearLayout.Orientation.Horizontal;
+            parent1layout.LinearAlignment = LinearLayout.Alignment.Bottom;
+            parent1layout.CellPadding = new Size(20, 50);
+            parent1.Layout = parent1layout;
+            root.Add(parent1);
+
+            parent2 = new View()
+            {
+                Position = new Position(500, 500),
+                Size = new Size(700, 70)
+            };
+            var parent2layout = new LinearLayout();
```

Comment:
if you dont use layout again, don't need to make variable

Reply:
Fixed